chsdi/templates/htmlpopup/shop_product.mako

```diff
@@ -0,0 +1,31 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+   lang = lang if lang in ('fr','it','en') else 'de'
+   fid = c['featureId']
+   webDavHost = request.registry.settings['webdav_host']
+   scale_str = str(c['attributes']['scale'])
+   n = ""
+   while len(scale_str)>3:
+     scale_prov = int(scale_str)/1000
+     n = n + "'000"
+     scale_str = str(scale_prov)
+   scale = "1:" + scale_str + n
+   price_float = c['attributes']['price']/100.0
+   price_2dec = format(price_float, '.2f')
+   price = "CHF " + str(price_2dec)
```

Comment:
same here, an helper function would make sense as well.
`formatPrice`
Provided a float it would return a nicely formatted string.